src/viur/core/render/vi/__init__.py

```diff
@@ -141,6 +141,8 @@ def canAccess(*args, **kwargs) -> bool:
 
 @exposed
 def index(*args, **kwargs):
+    if args or kwargs:
+        raise errors.NotFound
```

Comment:
Isn't it required to instantiate this?
```suggestion
        raise errors.NotFound()
```

Reply:
That's implicit done.


> The sole argument to [raise](https://docs.python.org/3.11/reference/simple_stmts.html#raise) indicates the exception to be raised. This must be either an exception instance or an exception class (a class that derives from [BaseException](https://docs.python.org/3.11/library/exceptions.html#BaseException), such as [Exception](https://docs.python.org/3.11/library/exceptions.html#Exception) or one of its subclasses). If an exception class is passed, it will be implicitly instantiated by calling its constructor with no arguments:
> 
> ```python
> raise ValueError  # shorthand for 'raise ValueError()'
> ```

_~ from the [Python docs](https://docs.python.org/3.11/tutorial/errors.html?highlight=shorthand#raising-exceptions)_

Reply:
OK thanks for the info, but to me it always looks wrong. The next time I come over this line I would correct it. Therefore, we just fix this.